content/en/integrations/guide/azure-portal.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Datadog in the Azure Portal
+kind: guide
+further_reading:
+  - link: "integration/azure"
+    tag: "Documentation"
+    text: "Azure integration"
+  - link: "https://www.datadoghq.com/blog/azure-datadog-partnership"
+    tag: Blog
+    text: "Microsoft partnership embeds Datadog natively in the Azure portal"
+---
+
+{{< site-region region="us,eu,gov" >}}
+<div class="alert alert-warning">
+  This guide is for Datadog customers using US3 with Azure.
+</div>
+{{< /site-region >}}
+
+This guide is for managing the integration between Azure and Datadog in the Azure portal using the Datadog resource. The Datadog resource in Azure represents the connection between a Datadog organization and an Azure subscription. [Create a Datadog resource][1] in Azure before proceeding with this guide.
```

Comment:
@ruthnaebeck The "Create a Datadog Resource" link should direct to the azure marketplace page here: https://docs-staging.datadoghq.com/ruth/docs-2083/integrations/guide/azure-marketplace/?tab=create#create-datadog-resource

currently its sending to the main integration page which is no good.